### PR TITLE
[6892] Add error pages (404, 500) to Wordify BP

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/6892

### CODE REVIEW

**Comment by jvega190** [comment]
@rart here are some screenshots for review:
Screenshot from 2024-06-18 15-06-50
404:
Screenshot from 2024-06-18 15-10-03
500:
Screenshot from 2024-06-18 15-10-47

File: templates/web/errors/layout.ftl [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  </head>
+  <body>
+    <div class="wrap">
+      <p>${errorCode}</p>
+      <h1 class="title">${errorTitle}</h1>
+      <p>${errorMessage}</p>
+      <a href="/" class="btn btn-primary rounded">Go back home</a>
+    </div>
+
+    <script defer src="/studio/static-assets/scripts/craftercms-xb.umd.js"></script>
```
Comment (jvega190) [inline templates/web/errors/layout.ftl]: @rart I'm thinking that this should be added to editorial error pages, what do you think? Please let me know and I can create a ticket

Comment (rart) [reply templates/web/errors/layout.ftl]: Yes, Editorial and any other BP with error pages.

**Review by rart — CHANGES_REQUESTED** [review]
Would it be possible to preserve header/footer?

File: templates/web/errors/layout.ftl [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<#macro layout
```
Comment (rart) [inline templates/web/errors/layout.ftl]:
```suggestion
<#macro errorPageLayout
```

Comment (rart) [reply templates/web/errors/layout.ftl]: Rename file too

Comment (rart) [reply templates/web/errors/layout.ftl]: Apply changes to DCO site & add screenshots for that one too pls

**Review by rart — CHANGES_REQUESTED** [review]

**Comment by jvega190** [comment]
> Would it be possible to preserve header/footer?

Both header and footer need the model for it to render (social media items, about us content, quick links, copyright text)

File: templates/web/errors/layout.ftl
Comment (rart) [inline templates/web/errors/layout.ftl:1]: Why are there two FTLs?

**Review by rart — CHANGES_REQUESTED** [review]

File: templates/web/errors/layout.ftl
Comment (jvega190) [reply templates/web/errors/layout.ftl:1]: That's weird since I renamed the file, removing this one

**Review by rart — APPROVED** [review]

**Review by sumerjabri — APPROVED** [review]

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Please update the version numbers in craftercms-plugin.yaml. If this doesn't break compatibility, then just bump the BP version number.

**Comment by jvega190** [comment]
@sumerjabri version updated, compatibility stays the same

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**